Orange/__init__.py

```diff
@@ -21,6 +21,8 @@
                  'evaluation', 'misc', 'preprocess', 'projection', 'regression',
                  'statistics', 'version', 'widgets']:
     globals()[mod_name] = _LazyModule(mod_name)
+    
+from Orange import modelling
```

Comment:
This should be added to the list of modules above instead of importing it directly.
It concerns some lazy importing magic (not going into details in this comment).